<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParallelCoordinatesVolumeClass</class>
 <widget class="QMainWindow" name="ParallelCoordinatesVolumeClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>640</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>ParallelCoordinatesVolume</string>
  </property>
  <property name="windowIcon">
   <iconset resource="parallelcoordinatesvolume.qrc">
    <normaloff>:/ParallelCoordinatesVolume/Resources/colour_line.png</normaloff>:/ParallelCoordinatesVolume/Resources/colour_line.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="lLayout" stretch="3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="16,6">
        <item>
         <widget class="QWidget" name="parallelArea" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,2,2">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,25">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="l1">
                 <property name="font">
                  <font>
                   <family>黑体</family>
                   <pointsize>10</pointsize>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="lineWidth">
                  <number>5</number>
                 </property>
                 <property name="text">
                  <string>文件</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_7">
                 <property name="spacing">
                  <number>1</number>
                 </property>
                 <property name="topMargin">
                  <number>1</number>
                 </property>
                 <property name="bottomMargin">
                  <number>1</number>
                 </property>
                 <item>
                  <widget class="QToolButton" name="addNodeFile">
                   <property name="text">
                    <string>...</string>
                   </property>
                   <property name="icon">
                    <iconset resource="parallelcoordinatesvolume.qrc">
                     <normaloff>:/ParallelCoordinatesVolume/Resources/add_to.png</normaloff>:/ParallelCoordinatesVolume/Resources/add_to.png</iconset>
                   </property>
                   <property name="iconSize">
                    <size>
                     <width>16</width>
                     <height>16</height>
                    </size>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="parallelLayoutFile">
                 <property name="spacing">
                  <number>6</number>
                 </property>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>3</number>
               </property>
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,25">
                 <property name="spacing">
                  <number>3</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="font">
                    <font>
                     <family>黑体</family>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>颜色</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_8">
                   <item>
                    <widget class="QToolButton" name="addNodeColor">
                     <property name="text">
                      <string/>
                     </property>
                     <property name="icon">
                      <iconset resource="parallelcoordinatesvolume.qrc">
                       <normaloff>:/ParallelCoordinatesVolume/Resources/add_to.png</normaloff>:/ParallelCoordinatesVolume/Resources/add_to.png</iconset>
                     </property>
                    </widget>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QHBoxLayout" name="parallelLayoutColor"/>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,25">
               <property name="spacing">
                <number>3</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="font">
                  <font>
                   <pointsize>10</pointsize>
                  </font>
                 </property>
                 <property name="text">
                  <string>透明度</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_9">
                 <item>
                  <widget class="QToolButton" name="addNodeAlpha">
                   <property name="text">
                    <string/>
                   </property>
                   <property name="icon">
                    <iconset resource="parallelcoordinatesvolume.qrc">
                     <normaloff>:/ParallelCoordinatesVolume/Resources/add_to.png</normaloff>:/ParallelCoordinatesVolume/Resources/add_to.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="parallelLayoutAlpha"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QWidget" name="ViewArea" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <layout class="QVBoxLayout" name="viewStack"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="parallelcoordinatesvolume.qrc"/>
 </resources>
 <connections/>
</ui>
